sql/mysql/Positive-Technologies/MySqlLexer.g4

```diff
@@ -138,6 +138,7 @@ LINEAR:                              'LINEAR';
 LINES:                               'LINES';
 LOAD:                                'LOAD';
 LOCK:                                'LOCK';
+LOCKED:                              'LOCKED';
```

Comment:
Should it be added to [keywordsCanBeId](https://github.com/antlr/grammars-v4/blob/master/sql/mysql/Positive-Technologies/MySqlParser.g4#L2591)?